tests/integration-tests/tests/ultraserver/test_gb200/test_gb200/91_nvidia_imex_prolog.sh

```diff
@@ -77,8 +76,8 @@ function get_ips_from_node_names() {
   ${SCONTROL_CMD} -ao show node "${_nodes}" | sed 's/^.* NodeAddr=\([^ ]*\).*/\1/'
 }
 
-function get_dna_parameter() {
-  jq -r ".cluster.${1}" "${DNA_JSON_FILE}"
+function get_compute_resource_name() {
+  echo "${2}" | sed -E "s/${1}(.+)-[0-9]+$/\1/"
```

Comment:
[CodeStyle, minor, not blocking] It is unclear what we expect as 1st and 2nd parameter of this function. I needed to look at how it was used below to understand it (with sed patterns it is always a bit cryptic) 
What about using local variable names to disambiguate arguments or simply add a comment?